### ISSUE TITLE
How to generate php from .proto files

### ISSUE THREAD

**Issue by lukebakken** [basho/riak-phppb-client#6]
In PR #5 I'm in the process of adding HLL support. However, I can't seem to figure out how the protobuf lib in `vendor/allegro/protobuf` can be used to parse the `.proto` files into PHP files.

I'm following the instructions [here](https://github.com/allegro/php-protobuf#usage), but `protoc-gen-php.php` is not present in `vendor/allegro/protobuf` after doing the process documented [here](https://github.com/basho/riak-phppb-client#composer-install). If I clone `allegro/php-protobuf` and follow their instructions, the file is present. I assume this has something to do with the fork we use in `riak-phppb-client` (https://github.com/christophermancini/php-protobuf")

My plan is to document and automate this process in a `Makefile`.

Issue #4 is blocked by this at the moment.

Thanks!

**Comment by lukebakken** [comment]
Currently getting this when trying to run `protoc`:

```
$ PATH="$HOME/opt/php-5.6.24/bin:$PATH" php ./vendor/bin/protoc --help

PHP Fatal error:  Cannot redeclare ProtobufParser::$verbose in /home/lbakken/Projects/basho/riak-phppb-client/vendor/allegro/protobuf/ProtobufCompiler/ProtobufParser.php on line 86

Fatal error: Cannot redeclare ProtobufParser::$verbose in /home/lbakken/Projects/basho/riak-phppb-client/vendor/allegro/protobuf/ProtobufCompiler/ProtobufParser.php on line 86
```

**Comment by lukebakken** [comment]
@christophermancini progress!

```
$ PATH="$HOME/opt/php-5.6.24/bin:$PATH" php ./vendor/bin/protoc --help

--help issue with proto file
USAGE: protoc [options] message.proto

  -d, --destination                 Destination folder for generated classes [default="./"]
  -n, --use-namespaces              Use native PHP namespaces [default="false"]
  -p, --filename-prefix [PREFIX]    Specify a prefix for generated file names [default=""]
  --psr                             Output class files in a psr-4 directory structure [default="false"]
  --package                         Manually define proto file package [default=""]
```

**Comment by lukebakken** [comment]
Addressed in 57f78c36ce94b36062fc3378fda2a50c006c0128
